openspec/changes/aoe-trash-retention-10d/specs/agent-manager/spec.md

```
@@ -0,0 +1,56 @@
+## RENAMED Requirements
+
+- FROM: `### Requirement: AoE configuration is chezmoi-managed at `~/.agent-of-empires/config.toml``
```

markdownlint-cli2 (0.23.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)